webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js

```diff
@@ -21,19 +22,72 @@ SignatureReport.CorrelationsTab = function (tabName) {
 
 SignatureReport.CorrelationsTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
 
+SignatureReport.CorrelationsTab.prototype.loadControls = function() {
+    var self = this;
+
+    var defaultProduct = $('#mainbody').data('default-product');
+    var defaultChannel = $('#mainbody').data('default-channel');
```

Comment:
I've fixed this with https://github.com/mozilla/socorro/pull/3531/commits/d22404b1ab5ef934f6d5987309fa24697ba578d2.